Makefile

```diff
@@ -16,14 +16,7 @@ override CLUSTERS_ARGS += $(CLUSTER_SETTINGS_FLAG)
 override DEPLOY_ARGS += $(CLUSTER_SETTINGS_FLAG)
 override E2E_ARGS += cluster1 cluster2 cluster3
 override UNIT_TEST_ARGS += test/e2e
-
-# Process extra flags from the `using=a,b,c` optional flag
-
-ifneq (,$(filter helm,$(_using)))
-override DEPLOY_ARGS += --service_discovery --deploytool_submariner_args '--set lighthouse.image.repository=localhost:5000/lighthouse-agent,lighthouse.image.tag=local,lighthouseCoredns.image.repository=localhost:5000/lighthouse-coredns,lighthouseCoredns.image.tag=local'
```

Comment:
Removing this will cause helm jobs to run with images from quay, thus not testing the latest change

Reply:
Those specific variables don't exist in the helm charts.  They probably did at some point. We now use 

```
  --set submariner.images.repository="localhost:5000" \
  --set submariner.images.tag="local"
```

in the deploy_helm script which I believe covers LH images as well. Unless I'm  missing something...

Reply:
Ah OK since helm charts were changed and now deploy operator which in turn deploys everything, so hopefully it indeed deploys the locally built images and not ones from Quay :)

Reply:
yeah I checked the operator code too - the submariner controller passes the `Submariner` object's `repository` and `version` fields to the `ServiceDiscovery` object and the service discovery controller uses those to build the image paths.